src/Directory.Build.props

```diff
@@ -5,6 +5,7 @@
     <!-- Include PDB in the built .nupkg -->
     <AllowedOutputExtensionsInPackageBuildOutputFolder
     >$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <DisableBuildTelemetry>true</DisableBuildTelemetry>
```

Comment:
Personally, I prefer the current name. Having it be a 'negative' phrase It makes it very clear that this is opt-out, and that by _default_, we're sending build telemetry.

Whereas naming it something positive (IMO) implicitly makes it seem more like it's opt-in. At least without having to go read some defaults somewhere.

You are completely correct that most don't have negative prefixes, but it's also not completely taboo, since there are 2 which start with negative prefixes.

----

Either way, I think this can be left for a second phase of refinement.